
# BIO613: DNA Sequence Analysis Project

## Author
* **Student:** Group (Git & Github) 
* **Course:** BIO-613 Programming for Bioinformatics
* **Institution:** National University-Sudan
* **Date:** September 2026

In [1]:
from Bio.Seq import Seq

dna = Seq("ATGGCGCGCTTA")
gc_content = (dna.count("G") + dna.count("C")) / len(dna) * 100
print("DNA Sequence:", dna)
print(f"GC Content: {round(gc_content, 2)}%")

DNA Sequence: ATGGCGCGCTTA
GC Content: 58.33%


In [2]:
at_content = (dna.count("A") + dna.count("T")) / len(dna) * 100

print("DNA Sequence:", dna)
print(f"AT Content: {round(at_content, 2)}%")

DNA Sequence: ATGGCGCGCTTA
AT Content: 41.67%


## Conclusion
The GC content of the target sequence was successfully calculated. The sequence exhibits a GC ratio of 58.33%, which reflects its thermal stability profile in DNA strands.